chemprop/cli/predict.py

```diff
@@ -186,7 +201,13 @@ def main(args):
         p_atom_descs=args.atom_descriptors_path,
         **featurization_kwargs,
     )
-    logger.info(f"test size: {len(test_data)}")
+
+    if n_components == 1:
```

Comment:
You could combine this logic check with the one on line 228 by waiting to pull test_data out of a list with something like:
```
logger.info(f"test size: {len(test_data[0])}")
...
test_dsets = [make_dataset(d, args.rxn_mode) for d in test_data]
if n_componenets == 1:
    test_dset = test_dsets[0]
else:
    test_dset = data.MulticomponentDataset(test_dsets)
```
not necessary, but a little simpler I think

Reply:
Thanks! I like it!